src/ops.rs

```diff
@@ -1,3 +1,3 @@
-mod oracle;
+pub mod oracle;
```

Comment:
define these in `mod.rs` in ops dir instead of this file

Reply:
I think that's a better way, I will update the code.

Reply:
According to the note of https://doc.rust-lang.org/reference/items/modules.html#module-source-filenames, we should stay unchanged.